tests/mypy/outputs/v1.0.1/plugin_success.txt

```diff
@@ -3,3 +3,4 @@
 136: error: Unexpected keyword argument "description" for "AddProject"  [call-arg]
 278: error: Unexpected keyword argument "foo" for "MyDataClass"  [call-arg]
 278: error: Unexpected keyword argument "bar" for "MyDataClass"  [call-arg]
+292: error: Unused "type: ignore" comment
```

Comment:
Can we just remove the unused type ignore?

Reply:
In principle it would be better if we could detect that this wasn't an untyped field and handle it properly, but I think that would amount to some complex type analysis to distinguish between `x = <some_function_that_returns_a_classmethod>()` and `x = <some_function_that_returns_an_int>()`. And, as noted in the original issue, I believe you can resolve this by giving it a proper annotation anyway like `classmethod` or whatever. (So I think it's okay to have the plugin raise that error.)

Reply:
That’s more than enough background explanation, thank you for digging and being thorough!